README.md

```diff
@@ -46,6 +46,14 @@ Install via pip with
 pip install git+https://github.com/CDCgov/PyRenew@main
 ```
 
+## Existing Models
+
+At present, the following models, which exist in external repositories, are written using PyRenew:
+
+`pyrenew-covid-wastewater` (see [here](https://github.com/CDCgov/pyrenew-covid-wastewater)): _Models and infrastructure for forecasting COVID-19 hospitalizations using wastewater data with PyRenew._
+
+`pyrenew-flu-light` (see [here](https://github.com/CDCgov/pyrenew-flu-light/)): _An instantiation in PyRenew of an [Epidemia](https://imperialcollegelondon.github.io/epidemia/) influenza forecasting model used in the 2023-24 respiratory season._
+
```

Comment:
```suggestion
## Models implemented with PyRenew

- [CDCgov/pyrenew-covid-wastewaterpyrenew-covid-wastewater](https://github.com/CDCgov/pyrenew-covid-wastewater): _Models and infrastructure for forecasting COVID-19 hospitalizations using wastewater data with PyRenew._
- [CDCgov/pyrenew-flu-light](https://github.com/CDCgov/pyrenew-flu-light/): _An instantiation in PyRenew of an [Epidemia](https://imperialcollegelondon.github.io/epidemia/) influenza forecasting model used in the 2023-24 respiratory season._

```

Reply:
What I've added is very close to what you want, which I can see the merit of, save for the fact that "Models implemented with PyRenew" has first letters capitalized to keep on theme and DHM's later comment is taken into account.